backend/history-service/src/middleware/cors.ts

```diff
@@ -0,0 +1,25 @@
+import cors from "cors";
+
+// TODO: Add production site to allowed origins
+const allowedOrigins = ["http://localhost:3000"];
```

Comment:
Just to let you know, after we deploy to the cloud, we will need to add amplify URL to origin list, so I change to this and let github actions to handle it instead.
```
const allowedOrigins =
  process.env.CORS_ALLOWED_ORIGINS
    ? process.env.CORS_ALLOWED_ORIGINS.split(",")
    : ["http://localhost:3000"];
```

Reply:
Sure, I will update this!